rspirv/tests/test_data.rs

```diff
@@ -0,0 +1,48 @@
+use rspirv::{
+    binary::{Assemble as _, Disassemble as _},
```

Comment:
Does this serve a purpose? I think it's better not to have the `as` if it doesn't.

Reply:
I find this to be a good way to use what the traits offer *without* including the trait names themselves into the visible space. I.e. it's a more clear indication of the intent here.